spec/bump/commands/preview_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "spec_helper"
+
+describe Bump::CLI::Commands::Preview do
+  it 'calls the server and display the link on success' do
+    stub_and_return('previews/post_success.http')
+
+    expect do
+      new_command.call(file: __FILE__, format: 'yaml')
```

Comment:
I don't get the `__FILE__` here 🤔 Can you pls clarify?

Reply:
Oh, I think I was lazy here: I needed a real file to make `open` work. I'd better stub the `open` method.